utils/fileupload.js

```diff
@@ -0,0 +1,83 @@
+
+export function fileUpload( files, setAttributes ) {
```

Comment:
Can we add some JSDocs to these two functions? 
I wonder if we can test them too, maybe by mocking `wp.api.models`, I understand it's not straightforward tough